dev/intro.ftd

```diff
@@ -0,0 +1,152 @@
+
+-- ftd.type heading-hero-bold: $fpm.type.heading-hero.font
+desktop: $fpm.type.heading-hero.desktop
+mobile: $fpm.type.heading-hero.mobile
+xl: $fpm.type.heading-hero.xl
+weight: 700
+
+-- ft.page: `intro` and it's Usage
+
+`-- intro` can be used as part of blog posts.
+
+-- ft.code: Example code for `-- intro`
+lang: ftd
+
+\-- intro:
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minimium.
+
+
+
+
+-- intro:
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minimium.
+
+
+
+
+-- ftd.column intro:
+body body:
+ftd.color backgroundcolor: $fpm.color.main.background.base
+width: fill
+
+
+
+--- intro-desktop:
+if: not $ft.is-mobile
+body: $body
+backgroundcolor: $backgroundcolor
+
+--- intro-mobile:
+if: $ft.is-mobile
+body: $body
+backgroundcolor: $backgroundcolor
+
+
+-- ftd.column intro-desktop:
+body body:
+ftd.color backgroundcolor: $fpm.color.main.background.base
+width: fill
+background-color: $backgroundcolor
+margin-top: $fpm.space.space-5
+margin-bottom: $fpm.space.space-6
+
+
+--- post-subtitle:
+body: $body
+
+
+-- ftd.column intro-mobile:
+body body:
+ftd.color backgroundcolor:
+width: fill
+background-color: $backgroundcolor
+
+
+
+--- post-subtitle:
+body: $body
+
+
+-- ftd.row post-subtitle:
+width: fill
+body body:
+max-width: 1056
+
+--- post-subtitle-desktop:
+if: not $ft.is-mobile
+body: $body
+
+--- post-subtitle-mobile:
+if: $ft.is-mobile
+body: $body
+
+
+-- ftd.row post-subtitle-desktop:
+width: fill
+margin-top: $fpm.space.space-9
+margin-bottom: $fpm.space.space-9
+body body:
+max-width: 1056
+
+--- ftd.row:
+width: fill
+border-left: 4
+padding-left: $fpm.space.space-8
+border-color: $fpm.color.main.border
+
+
+--- ftd.text: 
+text: $body
+color: $fpm.color.main.text
+role: $fpm.type.copy-large
+
```

Comment:
@priyanka9634  each new component which starts with `--` should have 10 lines gap between 2